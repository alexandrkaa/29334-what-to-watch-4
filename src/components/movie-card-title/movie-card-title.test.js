import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardTitle from './movie-card-title.jsx';


it(`<MovieCardTitle /> should match snapshot`, () => {
  const title = `Movie title`;
  const movieGenre = `Comedy`;
  const movieDate = 2020;
  const tree = renderer
    .create(
        <MovieCardTitle
          title={title}
          movieGenre={movieGenre}
          movieDate={movieDate}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
