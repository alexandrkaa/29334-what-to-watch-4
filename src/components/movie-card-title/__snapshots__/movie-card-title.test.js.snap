// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCardTitle /> should match snapshot 1`] = `
Array [
  <h2
    className="movie-card__title"
  >
    Movie title
  </h2>,
  <p
    className="movie-card__meta"
  >
    <span
      className="movie-card__genre"
    >
      Comedy
    </span>
    <span
      className="movie-card__year"
    >
      2020
    </span>
  </p>,
]
`;
